### ISSUE TITLE
converstaion suggestion in loneford

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#259]
```
just a thought, but to make the game play a little smoother/easier shouldn't
t the guard dressed in green outside the house where Telund is (the house next 
to the town sign) say i should report any suspicious activity to Kuldan?  he 
currently just says to report any suspicious activity.
```

Original issue reported on code.google.com by `blue...@gmail.com` on 21 Oct 2011 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
Created a Patch file for this.
```

Original comment by `SamuelPl...@gmail.com` on 19 Nov 2011 at 2:24

Attachments:
- [English Conversation Bugfix Kuldan.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-259/comment-1/English Conversation Bugfix Kuldan.patch)

**Comment by GoogleCodeExporter** [comment]
```
Looks good, thanks bluers0 & Samuel. Committed in r224.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 10:07
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.11**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
